bluetooth/nrf52/nrf52.ino

```diff
@@ -1,23 +1,36 @@
 
 #include "NRF52RxTx.h"
```

Comment:
it looks like this file is no longer used as it is copied to nrf52-rfm69/nrf52-rfm69.ino. @DelightedWaif please remove